conn_test.go

```diff
@@ -1740,49 +1735,6 @@ func TestProtocolVersionValidation(t *testing.T) {
 							},
 						},
 					},
```

Comment:
@at-wat We need check the MessageServerHello first. And if the version is invalid, there is no opportunity to receive the following message.

Reply:
@at-wat please run the workflows again. sorry for disturb.

Reply:
Hi @at-wat , test case failed again. However, it seems not due to the session resumption feature. Could you offer any help?